tests/lbs_py_override_unittest.py

```diff
@@ -252,6 +253,10 @@ def testBool(self):
         self.assertFalse(size2)
     #enddef
 
+    def testInt(self):
+        # just to make sure Size can be correctly interpreted as int in python 3.10
+        ctypes.c_int(Size("1 GiB"))
```

Comment:
Shouldn't this rather test `int(Size("1 GiB"))`?

Reply:
No, `int()` still uses `__int__`, the problem is that having `__int__` no longer means that `Size` "can be interpreted as integer". `float` also has `__int__` and it's enough for `int(1.0)` to work, but `ctypes.c_int(1.0)` gives `TypeError`. I want to test that `__index__` works (and to be notified when/if Python decides to change this again).

Reply:
Makes sense.